cds/modules/deposit/static/templates/cds_deposit/types/project/form.html

```diff
@@ -169,7 +169,7 @@ <h4>Record restrictions</h4>
                       </select>
                       <div ng-init="$ctrl.selectedRestricted=$ctrl.cdsDepositCtrl.record._access.read"  class="restricted mt-20"  ng-show="$ctrl.permissions ==='restricted'">
                         <label class="control-label ng-binding">
-                          Choose who has access to the record
+                          Choose who has edit access to the record
```

Comment:
I am not sure this is correct:
* Project restrictions: who can see the published record
* Share tab: who can edit the upload

Maybe better to find better words to eliminate this confusion and change both?